### ISSUE TITLE
Review the syntax of a WSL command

### ISSUE THREAD

**Issue by javiereguiluz** [symfony/symfony-docs#19951]
Here: https://symfony.com/doc/current/setup/symfony_server.html#enabling-tls

It's rendered like this:

![image](https://github.com/symfony/symfony-docs/assets/73419/6f8cff2d-e019-47e7-9bab-ad21201ea634)

But I don't know if the real command is:

```
explorer.exe \`wslpath -w $HOME/.symfony5/certs\`
```

or

```
explorer.exe `wslpath -w $HOME/.symfony5/certs`
```

We could move this to a `.. code-block:: terminal` inside the `tip` block.

**Comment by wouter90** [comment]
you need the command without escaping the backticks. With them it will open explorer but not the right folder :)

**Comment by javiereguiluz** [comment]
Closing as fixed in #19956.
